### DISCUSSION TITLE
How to specify "channels" for method "Equalize()"

### DISCUSSION THREAD

**Discussion by ew16** [dlemstra/Magick.NET#1236] [Help] [answered]
Hello,

I'm trying to convert some of my scripts into Magick.Net.
E.g.: convert in.tif -channel RGB -equalize -negate -normalize -quality 70 out.jpg

The -negate statement can be transformend into : image.Negate(Channels.RGB);
But what to do with -equalize : image.Equalize(Channels.RGB); is not working - Equalize has no Channels paramter.

How can I specify the channels for Equalize ?
Thank you
Erwin

**Comment by ew16** [comment]
Some additional infos:
it's the sync flag which is used/evaluated by -equalize
You get different results from -channels RGB -equalize and -channels RGB,sync -equalize.
Where/How can I set or reset the sync-flag for the Equalize-method in Magick.NET ?
Thanks again.

**Comment by dlemstra** [comment, marked as answer]
It looks like this overload is missing. I will make sure that this gets added in the next release.

**Reply by dlemstra** [reply]
This was added in version 12.2.0.
